app/helpers/payment_details_helper.rb

```diff
@@ -30,7 +30,7 @@ def edit_payment_details(merchant_account = site_account)
   end
 
   def merchant_countries
-    ActiveMerchant::Country::COUNTRIES.map{|c| c[:name] }
+    ActiveMerchant::Country::COUNTRIES.map{|c| [c[:name], c[:alpha2]] }
```

Comment:
I haven't checked after this change, but just in case, check this: [THREESCALE-6529 - Fix country format/code incompatibility in stripe](https://issues.redhat.com/browse/THREESCALE-6529)